### ISSUE TITLE
code clarity

### ISSUE THREAD

**Issue by woldier** [ChenjieCSU/MemoryAdaptNet-master#7]
https://github.com/RS-CSU/MemoryAdaptNet-master/blob/365e3672f9f9f36a82738ed4aa7875a5ed642823/network/utils.py#L88C1

Old code that does not specify dim may cause the `softmax` function to not work as expected.

```python
torch.allclose(F.softmax(preds_stage2.detach(), dim=1), F.softmax(preds_stage2.detach()))
```
